<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QWidget" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QLineEdit" name="le_ip">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>221</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_port">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>270</y>
     <width>221</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>12345</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_message">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>491</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Mensaje recibido ^^</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>500</y>
     <width>147</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_connect">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>500</y>
     <width>147</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>Conectar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_ip">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>151</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Direccion IP</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
